lib/LoginStrategy.ts

```diff
@@ -351,6 +425,27 @@ export class LoginStrategy extends BaseStrategy {
     );
   }
 
+  /**
+   * extracts siwe message and signature either from request body or query
+   * @param req
+   * @returns {string} siwe object - message and signature
+   */
+  extractSiwe(req: Request): SiweMessageReqPayload {
```

Comment:
Can be shortened by removing `Message`, IMO
```suggestion
  extractSiwe(req: Request): SiweReqPayload {
```